roles/openshift_autoheal/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Namespace and name:
+openshift_autoheal_name: autoheal
```

Comment:
I'm will need to relay on this name not changing in the service monitor definition

Reply:
and also the namespace being `openshift-autoheal` do we support changing them @jhernand ?

Reply:
This is our call, I guess. Do we want these configurable or not?